app/views/alchemy/admin/attachments/_filter_bar.html.erb

```diff
@@ -0,0 +1,32 @@
+<% params_to_keep = [:tagged_with, :q, :content_id, :options] %>
+
+<div id="filter_bar">
+  <h2><%= Alchemy.t('Filter') %></h2>
```

Comment:
Could this be "File type" instead of "Filter"? That would be more explicit as to what the select box actually does.